src/grammar/statement/select.rs

```diff
@@ -80,7 +80,11 @@ impl Statement for Select {
             buffer.push("*");
         }
         buffer.push("FROM");
-        buffer.push(format!("`{}`", some!(self.table)));
+        let mut do_comma = false;
+        for table in &self.table{
+            if do_comma{ buffer.push(",");} else{do_comma=true;}
```

Comment:
Please rewrite using `enumerate`, in which case you’ll know the index of the current table in the array.

Reply:
Please also format the code properly.

Reply:
I'm sorry, I got stuck with this macro error. I don't understand why.
```rust
buffer.push(format!("`{}`", some!(self.table.first()) )
unexpected end of macro invocation
```

Reply:
Oh, because the `some!` macro requires that the argument should be a property of the struct `($this:ident.$field:ident) ` . The `some!` macro doesn't fit for extracting the first table of the `self.table`.